linter/whitelists/de.txt

```diff
@@ -398,4 +398,8 @@ Suchabos
 mobilfähig
 mobilfähige
 Formvalidation
-OriginalFileName
\ No newline at end of file
+OriginalFileName
```

Comment:
These are not required. The things you add to `default.txt` are valid for all languages. So you can remove the ones you added in `default.txt` in both `de.txt` as well as `en.txt`.